src/loader-recorder-v2.ts

```diff
@@ -365,6 +366,43 @@ function prepareRequest(
     return requests
 }
 
+function _checkForCannotReadBody(res: Response): string | null {
+    if (res.headers.get('Transfer-Encoding') === 'chunked') {
+        return 'Chunked Transfer-Encoding is not supported'
+    }
+
+    const contentType = res.headers.get('Content-Type')
+    if (contentType && (contentType.startsWith('video/') || contentType.startsWith('audio/'))) {
```

Comment:
I think `multipart` can contain anything and might be a problem, and `application` is always (or almost always) binary, we probably don't want that either?

Reply:
yep, I'm slightly copying other repos as well which seem to go for a "close enough for rock and roll" approach

🤔

Reply:
I think we can ignore `application` here as well and let the `multipart` throw if needed, otherwise, we might be losing what we can read, just food for thought.

Reply:
yeah, I've refactored to a more explicit deny list so that we have somewhere we can easily add new values if folk report them 👍